deploy/examples/cluster.yaml

```diff
@@ -166,6 +166,10 @@ spec:
 #    osd:
 #    cleanup:
 #    prepareosd:
+# clusterMetadata annotations will be applied to only `rook-ceph-mon-endpoints` configmap and the `rook-ceph-mon` and `rook-ceph-admin-keyring` secrets.
+# And clusterMetadata annotations will not be merged with `all` annotations.
+#     clusterMetadata:
```

Comment:
nit: looks like there is an extra space to be consistent with the indentation of "prepareosd" above.
```suggestion
#    clusterMetadata:
```